virt/post_installation_configuration/virt-post-install-network-config.adoc

```diff
@@ -0,0 +1,53 @@
+:_content-type: ASSEMBLY
+[id="virt-post-install-network-config"]
+= Post-installation network configuration
+include::_attributes/common-attributes.adoc[]
+:context: virt-post-install-network-config
+
+toc::[]
+
+By default, {VirtProductName} is installed with a single, internal pod network.
+
+After you install {VirtProductName}, you can install networking Operators and configure additional networks.
+
+[id="installing-operators"]
+== Installing networking Operators
+
+You can install the Kubernetes NMState Operator and SR-IOV Operator.
```

Comment:
The NMstaet operator should always be installed, it makes like easier.  Please explain the benefit, a simple "WHY", rather than just the factual, "you can do this".